centreon/src/Core/Security/ProviderConfiguration/Infrastructure/Logger/LoginLogger.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *  For more information : contact@centreon.com
+ */
+
+namespace Core\Security\ProviderConfiguration\Infrastructure\Logger;
+
+use CentreonUserLog;
+use Core\Security\ProviderConfiguration\Domain\LoginLoggerInterface;
+use Pimple\Container;
+
+class LoginLogger implements LoginLoggerInterface
+{
+    /**
+     * @var CentreonUserLog
+     */
+    private CentreonUserLog $logger;
+
+    /**
+     * @param Container $container
+     */
+    public function __construct(private Container $container)
+    {
+        $pearDB = $this->container['configuration_db'];
```

Comment:
This property seems to be not used elsewhere in the class : useless property
```suggestion
    public function __construct(Container $container)
    {
        $pearDB = $container['configuration_db'];
```